setup_bibliography.Rmd

```diff
@@ -0,0 +1,58 @@
+---
+title: "Bibliography Setup"
+author:
+  name: "Thomas Roppelt"
+  affiliation: "Friedrich-Alexander-Universität Erlangen-Nürnberg (FAU)"
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+bibliography: library.bib
+---
+
+```{r setup, echo = FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+## Bibliography Managers
+Bibliography managers are important to keep all the literature you work with well organized.
+This page will help you understand pros and cons of different bibliography managers as well as help you set up Zotero, a free and open-source bibliography manager.
+
+## Comparison
+There are many different bibliography managers available, e.g. Citavi, Colwiz, EndNote, JabRef, Mendeley, Zotero and ZoteroBib.
+Since we focus on open-source software, software like Citavi and EndNote are out of question.
```

Comment:
I think an alternative/similar angle might me (because some people might not be swayed by "just use FOSS" 😏):

1. strictly speaking, we can accept *any* reference manager that produces a clean `.bib`.
2. more broadly, you definitely want a reference manager that you can use for years/decades, *or* at least get your data out of. This kinda rules out proprietary software, which mostly use proprietary and weird databases.

Some conclusion, just different reasons.

Just a suggestion of course.

Reply:
Good point, I'll update the wording